packages/builders/src/index.ts

```diff
@@ -13,7 +13,6 @@ export * as TextInputAssertions from './components/textInput/Assertions.js';
 export * from './interactions/modals/Modal.js';
 export * as ModalAssertions from './interactions/modals/Assertions.js';
 
-export * from './components/selectMenu/BaseSelectMenu.js';
```

Comment:
Why not export it?
You should still be able to use `instanceof` with the base class.

Reply:
@vladfrangu

Reply:
We opted to keep this.